inspirehep/config.py

```diff
@@ -1188,3 +1188,9 @@ def _(x):
     "stat.OT": "Other",
     "stat.TH": "Other"
 }
+
+# Import local config file if it is present.
+try:
+    from inspirehep.config_local import *
```

Comment:
When you're developing locally. E.g.use of assets debug. Setting the magpie
or beard endpoints etc.

On Mon, 27 Jun 2016, 09:14 Javier Martin Montull, <notifications@github.com>
wrote:

> In inspirehep/config.py
> <https://github.com/inspirehep/inspire-next/pull/1251#discussion_r68531023>
> :
>
> > @@ -1188,3 +1188,9 @@ def _(x):
> >      "stat.OT": "Other",
> >      "stat.TH": "Other"
> >  }
> > +
> > +# Import local config file if it is present.
> > +try:
> > +    from inspirehep.config_local import *
>
> And where is the config_local file? In Puppet for example we are filling
> the inspirehep.cfg file that goes into var/instance. When would we use
> the config_local file?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/inspirehep/inspire-next/pull/1251/files/ae94900a7ca8c08f1ddb50f57244c5415da95919#r68531023>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AARPHI3XpgMJSmL2hI1x3JRNG0exHvAzks5qP3hXgaJpZM4I-UY7>
> .
>